### PR TITLE
Added basic user view

### PR DESCRIPTION
**Pull request by davidurschel**
Still need to write functionality to trigger opening this module as well as the back-end implementations

### CODE REVIEW

File: napkinapp/app/src/main/java/com/example/napkinapp/MainActivity.java [hunk 1]
```diff
@@ -23,7 +25,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // Load content fragment
         getSupportFragmentManager().beginTransaction()
-                        .replace(R.id.content_fragmentcontainer, new ListEventsFragment())
+                        .replace(R.id.content_fragmentcontainer, new ProfileFragment())
```
Comment (Sherali5543) [inline napkinapp/app/src/main/java/com/example/napkinapp/MainActivity.java:28]: revert this change. Screen will open up on listevents

File: napkinapp/app/src/main/java/com/example/napkinapp/fragments/HeaderFragment.java [hunk 2]
```diff
@@ -20,5 +20,6 @@ public HeaderFragment(){
     @Override
     public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
         return inflater.inflate(R.layout.main_header, container, false);
+
```
Comment (Sherali5543) [inline napkinapp/app/src/main/java/com/example/napkinapp/fragments/HeaderFragment.java:23]: what are you adding whitespace

**Review by Sherali5543 — CHANGES_REQUESTED** [review]
Gonna do a pull later today to review ui

File: napkinapp/app/src/main/java/com/example/napkinapp/MainActivity.java (hunk 1 above)
Comment (davidurschel) [reply napkinapp/app/src/main/java/com/example/napkinapp/MainActivity.java:28]: Yeah, we need to add the navigation functionality soon, just because it kind of exists at the top level, let me know when we can meet to discuss implementation details
